core/src/main/java/io/cucumber/core/feature/GluePath.java

```diff
@@ -29,6 +34,11 @@
  */
 public class GluePath {
 
+    private static final Logger log = LoggerFactory.getLogger(GluePath.class);
+
+    private static final Pattern FILESYSTEM_PATH_TO_PACKAGES = Pattern
+            .compile("/*src/(?:main|test)/(?:java|kotlin|scala|groovy)/(.*)");
```

Comment:
This pattern suggests that `/src/main/java/com/example` is a file system path to a package. But I think it is unlikely that the file system root is part of any project. So I don't think we have to warn users about this. They'll be warned did not find any glue.

The pattern also suggests that `src/main/java` is not a path to a package (it's the root package). Consider adding a test case for this.